integration/local-gadget/integration_test.go

```diff
@@ -0,0 +1,256 @@
+package main
+
```

Comment:
I think splitting this file into a per gadget file will help us (https://github.com/kinvolk/inspektor-gadget/issues/974). What about splitting the file now before merging this PR?